.github/workflows/porch.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Porch
+
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "site/**"
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+      - "site/**"
+
+jobs:
```

Comment:
Two things that will help us to analyze if we need to adjust the presubmit workflow (locally and in CI)
- How long does the porch specific tests take ? 
- What dependencies do a kpt contributor needs to install to run tests for porch

Reply:
Porch tests are about 5 mins (https://github.com/GoogleContainerTools/kpt/actions/runs/1867953046)

no dependencies needed.

The e2e tests will have mode where they run against a k8s cluster where they'd install Porch but that part is not yet implemented.

Reply:
ok.